test/e2e/page-objects/pages/dialog/dialog.ts

```diff
@@ -0,0 +1,20 @@
+import { Driver } from '../../../webdriver/driver';
+import { strict as assert } from 'assert';
+
+class GenericConfirmCancelDialog {
+  protected driver: Driver;
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async clickConfirm() {
+    await this.driver.clickElement({ text: 'Confirm', tag: 'button' })
```

Comment:
maybe we could add the selectors as variables above, like done in other classes?